specificatie/openapi.yaml

```diff
@@ -648,6 +828,94 @@ components:
               items:
                 $ref: '#/components/schemas/ZoekResultaatHal'
 
+    GeoJsonZoekOpdracht:
+      description: 'Zoekobject om te zoeken op geometrie.'
+      type: object
+      properties:      
+        geometrie:
+          $ref: 'http://schemas.opengis.net/ogcapi/features/part1/1.0/openapi/schemas/polygonGeoJSON.yaml'
+        spatialOperator:
+          type: string
+          description: 'Een uit de set van spatial operators die gedefinieerd zijn in ISO 19125-1'
+          enum:
+#            - bbox
+#            - equals
+#            - disjoint
+            - intersects
+#            - touches
+#            - crosses
+            - within
+            - contains
+#            - overlaps
+#            - beyond
+#            - dwithin
+        tolerantie:
+          type: number
+
+    AdressenZoekObject:
+      type: object
+      description: Een object waarin de benodigde zoek criteria worden opgegeven om naar adressen te zoeken.
+      properties:
+        _geo:
+          $ref: '#/components/schemas/GeoJsonZoekOpdracht'
+        bouwjaar:
+          description: Een aanduiding voor de minimale en/of maximale waarde voor een bouwjaar van een pand op het adres.
+          type: object
+          properties:
+            min:
+              type: integer
+              minimum: 0
+              maximum: 9999
+            max:
+              type: integer
+              minimum: 0
+              maximum: 9999
+
+    AdresseerbareObjectenZoekObject:
+      type: object
+      description: Een object waarin de benodigde zoek criteria worden opgegeven om naar adresseerbare objecten te zoeken.
+      properties:
+        _geo:
+          $ref: '#/components/schemas/GeoJsonZoekOpdracht'
+        statusAdresseerbaarObject:
+          type: array
+          minItems: 1
+          items:
+            $ref: '#/components/schemas/AdresseerbaarObjectStatus_enum'
+        geconstateerd:
+          type: boolean
+        oppervlakte:
+          type: object
+          properties:
+            min:
+              type: integer
+            max:
+              type: integer
+        gebruiksdoel:
+          type: array
+          minItems: 1
+          items:
+            $ref: '#/components/schemas/Gebruiksdoel_enum'
+        typeAdresseerbaarObject:
+          type: array
```

Comment:
@CathyDingemanse is hier behoefte aan zoeken op meerdere types verblijfsobjecten (bijvoorbeeld ligplaatsen + standplaatsen) in één vraag? Of is zoeken op één type genoeg?